tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/entity/FixResult.java

```diff
@@ -27,17 +29,25 @@
   /** The position at the query where the error occurs. */
   Position errorPosition;
 
+  /** Is query fixer confident about this fixing */
+  Boolean isConfident;
+
+  /** The reason why this fix is failed. It is only not null when the Status is FAILURE. */
```

Comment:
nit: fix 'failed'?

Reply:
Rephrase to "The detail of why the query fails to be fixed.".